### PR TITLE
Update Observation documentation:

### PR DESCRIPTION
**Pull request by MaxPhilips**
* Remove component-code parameter
* General tweaks for clarity

### CODE REVIEW

File: content/dstu2/observation.md [hunk 1]
```diff
@@ -19,14 +19,23 @@ Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied
...
- `date`            | Recommended to search by `code`, `component-code`, or `date` | [`date`]      | Date range into which the observation falls (effectiveDateTime). Single dates are treated as implied ranges. EG: `date=gt2014-09-24T12:00:00.000Z` `&date=lt2015-09-24T12:00:00.000Z`
- [`_count`]        | N                                                            | [`number`]    | The maximum number of results to return per page.
+ Name             | Required?         | Type          | Description
+------------------|-------------------|---------------|----------------------------------------------------------------------------------------------------------------------
+ `patient`        | This or `subject` | [`reference`] | The subject that the observation is about (if patient). Example: `patient=1316024`
+ `subject`        | This or `patient` | [`reference`] | The subject (Patient) that the observation is about. Example: `subject=Patient/1316024` or `subject:Patient=1316024`
+ `code`           | N                 | [`token`]     | The code of the observation type. Example: `code=http://loinc.org|3094-0,http://loinc.org|3139-3`
```
Comment (daliboz) [inline content/dstu2/observation.md]: indicate that this will also search component codes

Comment (MaxPhilips) [inline content/dstu2/observation.md]: Updated to "The code or component-code of the observation type." and added a note in the notes section:
https://github.com/cerner/fhir.cerner.com/pull/112/commits/7fb4b526a1903b41cf80258986f38549d039fe46

**Comment by jfrhong** [comment]
+1

**Comment by MaxPhilips** [comment]
Squashing and merging

**Pull request merged**